AnnoDesigner/Localization/LocalizeBinding.cs

```diff
@@ -51,4 +61,31 @@ public Localize(string key) : this()
             Key = key;
         }
     }
+
+    public class DynamicLocalize : MultiBinding
+    {
+        private static LocalizeConverter LocalizeConverter { get; } = new LocalizeConverter();
+
+        public string KeyPath
```

Comment:
This should be renamed to `Key` for consistency with `LocalizeBinding`. The parameter name in the constructor should be updated as well.

Reply:
KeyPath is more accurate, since it is a path to the translation key.

Reply:
I thought it was the key to the translations dictionary?

Reply:
No, for DynamicLocalize it is a path to the translation key, which will then be used to translate by dictionary.

Reply:
Essencially something like this: `<TextBlock Text={l:Localize Key={Binding Path=KeyPath}} />`, if Binding would support dependency properties.

Reply:
Ah - yep - I see that now.